Autopilot/AttitudeManager/Src/attitudeStateClasses.cpp

```diff
@@ -131,7 +131,7 @@ void sendToSafetyMode::execute(attitudeManager* attitudeMgr)
 {
     SendToSafety_error_t ErrorStruct;
     float *channelOut = OutputMixingMode::GetChannelOut();
-    for(int channel = 0; channel < 8; channel++) //we use up to 7
+    for(int channel = 0; channel < 8; channel++) // currently using channels 0-8
```

Comment:
7*. If it were <=, then it would be 8. Correct me if I am mistaken

Reply:
you are mistaken. We have 8 channels. 0-7